scripts/start.sh

```diff
@@ -48,6 +48,24 @@ sleep 5
 echo "Creating role bindings for principals"
 docker-compose exec tools bash -c "/tmp/helper/create-role-bindings.sh" || exit 1
 
+# Workaround for setting min ISR on topic _confluent-metadata-auth
+docker-compose exec kafka1 kafka-configs \
```

Comment:
Some of these CLI commands use port 12091, and some use 11091. Is that intentional?

Reply:
The reasoning was to differentiate:

1. the examples using 11091 are "best practice" and require security configs, whereas
2. the examples using 12091 are to the PLAINTEXT port and are workarounds

Reply:
Makes sense to me.